instrumentation/http-url-connection/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/httpurlconnection/HttpUrlConnectionInstrumentationModule.java

```diff
@@ -85,8 +89,13 @@ public static void methodEnter(
         @Advice.FieldValue("connected") boolean connected,
         @Advice.Local("otelHttpUrlState") HttpUrlState httpUrlState,
         @Advice.Local("otelScope") Scope scope,
-        @Advice.Local("otelCallDepth") CallDepth callDepth) {
+        @Advice.Local("otelCallDepth") CallDepth callDepth,
+        @Advice.Origin("#m") String methodName) {
 
+      if (methodName.equals("getResponseCode")) {
```

Comment:
It looks like for `getResponseCode()` instrumentation has nothing in common with the other instrumented methods -- consider extracting it to a separate advice class and just put another map entry in the `transformers()` map.

Reply:
Yeah, if you look at the earlier commits I did have a new field in the `HttpUrlState` class, but it is only a local and isn't shared with the other advice calls for the other methods.  I will see what it looks like with a new advice.

Reply:
I think it's an improvement.  Thanks for the nudge.